.test-infra/jenkins/job_PerformanceTests_PubsubIO_Python.groovy

```diff
@@ -50,7 +50,7 @@ def psio_test = [
     num_workers               : 5,
     autoscaling_algorithm     : 'NONE',  // Disable autoscale the worker pool.
     pubsub_namespace_prefix   : 'pubsub_io_performance_',
-    wait_until_finish_duration: 1000 * 60 * 10, // in milliseconds
+    wait_until_finish_duration: 1000 * 60 * 12, // in milliseconds
```

Comment:
Because tests now running on Runner V2 and with longer launch time (~6 min), with high probability the write pipeline gets time out at half way writing to pubsub. Then the read pipeline never receives expected number of elements and fails the matcher.

There is one success run (albeit error in cleanup): https://ci-beam.apache.org/job/beam_PerformanceTests_PubsubIOIT_Python_Streaming_PR/28/console

Other runs fails at matcher assert. Increased timeout and it should saffice. But need a seed job to take effect.